data-es6.js

```diff
@@ -11986,6 +12031,7 @@ exports.tests = [
     rhino17:     true,
     node012:     true,
     android40:   true,
+    edge13:      false,
```

Comment:
well, that's what Edge 13 latest tells me. Either that, or the test is faulty somehow.